llvm/utils/lit/tests/Inputs/shtest-cat/lit.cfg

```diff
@@ -0,0 +1,8 @@
+import lit.formats
+
+config.name = "shtest-cat"
+config.suffixes = [".txt"]
+config.test_format = lit.formats.ShTest()
+config.test_source_root = None
+config.test_exec_root = None
+config.substitutions.append(("%{python}", '"%s"' % (sys.executable)))
```

Comment:
Are you using python? If not, please remove.

Reply:
done